client/src/components/mapPopup.module.scss

```diff
@@ -15,6 +8,7 @@
   bottom: 12px;
   left: -50px;
   min-width: 280px;
+  overflow: scroll;
```

Comment:
so curious how this looks on different screen sizes haha

Reply:
Yes - Definitely true, we need an explicit mobile pass - I just filed #268 to make sure we do this